static/js/impala/suggestions.js

```diff
@@ -130,12 +130,14 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
             $results.filter('.visible').removeClass('visible');
             return;
         }
+        var urlVal = encodeURIComponent($self.val());
```

Comment:
> The ' doesn't cause a problem in the url and ends-up as %27 in the resulting href when inserted
Seems, it happens on JS Bin only.

Try `escape_(encodeURIComponent("'foo'"))` in Chrome DevTools.

If you enter `'foo'` into search box on AMO site browser will convert single quotes to %27 by itself.